cvat-core/src/server-proxy.js

```diff
@@ -223,6 +239,35 @@
                 }
             }
 
+            async function exportDataset(id, format) {
+                const { backendAPI } = config;
+                let url = `${backendAPI}/tasks/${id}/export?format=${format}`;
```

Comment:
@zhiltsov-max , I would prefer another way:

```suggestion
                let url = `${backendAPI}/tasks/${id}/dataset?format=${format}`;
```

`GET /tasks/${id}/dataset?format=${format}` looks very natural.